### DISCUSSION TITLE
feature request: a way to search for "which formula provides this executable"

### DISCUSSION THREAD

**Discussion by rdp** [Homebrew/discussions#905] [Everyday usage] [answered]
based on https://apple.stackexchange.com/q/193795/25085

It would be nice if "brew search strings" would inform one that it's in the binutils package.  Maybe formulae could have another option "provides these executables" to make things searchable like that?  Cheers!

**Comment by carlocab** [comment, marked as answer]
Not exactly what you're looking for, but close: https://github.com/Homebrew/homebrew-command-not-found

**Edit:** Actually, this does do what you're asking. Once you've tapped this repository, the command you want is `brew which-formula`.

**Reply by rdp** [reply]
Yeah that's it, thanks!  Might be nice if it were made somehow inline more.  Thanks!
